docs/Job-reference.md

```diff
@@ -5833,6 +5834,24 @@ job('example-2') {
 
 (since 1.26)
 
+### Active Choice Parameter
+
+```groovy
+activeChoiceParam(String paramName, Closure closure = null) {
+    description(String description)
+    visibleItemCount(int visibleItemCount)
```

Comment:
There is no UI option for this. Why does this need to be configured by the DSL then?